Hunting Queries/SecurityEvent/Discorddownloadinvokedfromcmdline.yaml

```diff
@@ -0,0 +1,31 @@
+id: e7dd442a-0af8-48eb-8358-9e91f4911849
+name: Discord download invoked from cmd line
+description: |
+  'This hunting query looks for hosts that have attempted to interact with the Discord CDN. This activity is not normally invoked from the command line and could indicate C2, exfiltration, or malware delivery activity.'
+requiredDataConnectors:
+  - connectorId: SecurityEvents
+    dataTypes:
+      - SecurityEvents
+tactics:
+  - Execution
+  - CommandAndControl
+  - Exfiltration
+relevantTechniques:
+  - T1204
+  - T1102
+  - T1567
+query: |
+    SecurityEvent
```

Comment:
Why are we duplicating this when we have imProcess for this above?

Reply:
Neither imRegistry or imProcess are built in parsers. There will be a large number of workspaces that can't currently leverage them directly so providing both options for greatest reach and impact.